src/plugin.js

```diff
@@ -0,0 +1,2 @@
+const plugin = require('ilp-plugin')()
```

Comment:
I don't think we need the `src/plugin` file anymore

Reply:
I use plugin in pay.js? I can instead just do require (ilp-plugin) though